.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"
```

Comment:
`excellent.gif`

Reply:
Looking forward to Bundler 2.2.0 to be released. It's been in RC for a long time...

The new version fixes the bug where calling `BUNDLE_WITH=group` touches this file (see https://github.com/rubygems/rubygems/issues/3708) which currently makes it confusing to run some of our workflows when this file is tracked because it changes it and people might not know whether to track the change or not.

Internal ref paaHJt-13C

🤞 it won't be long...